### DISCUSSION TITLE
SyntaxError: invalid syntax

### DISCUSSION THREAD

**Discussion by samfisher9610** [swar/Swar-Chia-Plot-Manager#1184] [General]
hello 
getting this error on swar plot manger 
I have no knowlage in cooding cant able to solve this 
plz help me to fix this 
any help will be appreciated

Traceback (most recent call last):
  File "manager.py", line 4, in <module>
    from plotmanager.library.utilities.commands import start_manager, stop_manager, view, json_output, analyze_logs
  File "/home/sam/Desktop/Swar-Chia-Plot-Manager-main/plotmanager/library/utilities/commands.py", line 15
    from plotmanager.library.utilities.print import print_view
                                           ^
SyntaxError: invalid syntax

**Comment by samfisher9610** [comment]
solved

**Comment by jaqbmecom** [comment]
como se resolvió?
